vuepress/docs/next/tutorials/create/pb/publish-project-bundle.md

```diff
@@ -155,13 +155,16 @@ ent prj install
 ent prj install --conflict-strategy=OVERRIDE
 ```
 
+  > **Note**
+  >
+  > In the event of a timeout or pod failure, refer to the errors captured by the quickstart-cm-deployment and plugin deployer logs. 
+
 ## Use the App Builder
-1. Log into the `App Builder`
+1. Log into your App Builder and select `Component Repository` in the upper right
 
-2. Select `Component Repository` in the upper right
+2. Find your bundle and select `Install`
 
-3. Find your bundle and select `Install`
+In this stage, the Entando Platform downloads and installs the Docker image for your microservice and installs your micro frontends into the Entando Application. You can add these micro frontend widgets to the page(s) of your choice.
 
-At this point the Entando platform will download and install the Docker image for your microservice and install the micro frontends into the Entando application. You can add those micro frontend widgets to the page(s) of your choice.
+You can leverage a page provided by Entando or follow our [page management tutorial](../../compose/page-management.md) to create your own page and/or page template. Please note that an application based on the Entando Blueprint expects a user to be authenticated, so you'll need to include the `keycloak_auth` UX fragment in your page template.
```

Comment:
capitalize page title? >>[Page Management tutorial]

Reply:
i was torn here... i can go either way. i kind of like it lowercase because it doesn't interrupt the sentence flow - it just reads as a regular sentence. i think we may be doing this both ways here and there but mostly capitalized? if you feel strongly about capitalization here i'll change it.

Reply:
It's not a big deal, but when I read things like this, I tend to scan until the link comes up to see if it's something I'm looking for, so I like bigger highlighting. It can go either way though: read as descriptive words for the link or the page title. I tend to capitalize if it is the page title.

Reply:
@jyunmitch agreed. let's do that.